content/v2.0/reference/api/client-libraries/python.md

```diff
@@ -90,16 +85,86 @@ from influxdb_client.client.write_api import SYNCHRONOUS
 bucket = "<my-bucket>"
 org = "<my-org>"
 token = "<my-token>"
+#variable to store the url of your local or InfluxDB Cloud instance
+url="<http://localhost:9999>"
 
 client = influxdb_client.InfluxDBClient(
-    url="http://localhost:9999",
+    url=url,
     token=token,
     org=org
 )
 
-client = influxdb_client.InfluxDBClient(url=url, token=token, org=org)
 write_api = client.write_api(write_options=SYNCHRONOUS)
 
 p = influxdb_client.Point("my_measurement").tag("location", "Prague").field("temperature", 25.3)
 write_api.write(bucket=bucket, org=org, record=p)
 ```
+## Query data from InfluxDB with Python
+
+To query data, instantiate the **query client**. 
+
+```python
+query_api = client.query_api()
+```
+
+Next, create a flux query. 
+
+```python
+query = ‘ from(bucket:"my-bucket")\
+|> range(start: -10m)\
+|> filter(fn:(r) => r._measurement == “my_measurement”)\
```

Comment:
`“my_measurement”` is using typographic (curly) quotes. Change to straight quotes.